pyvista/core/pointset.py

```diff
@@ -1,5 +1,5 @@
 """Sub-classes and wrappers for vtk.vtkPointSet."""
-import collections
+from collections.abc import Iterable
```

Comment:
Ah, nice, I didn't know they finally pulled the plug on this one. This was deprecated in 3.3! https://docs.python.org/3.9/library/collections.html